system_baseline/views/v1.py

```diff
@@ -246,6 +246,30 @@ def get_baselines(limit, offset, order_by, order_how, display_name=None):
     )
 
 
+@metrics.baseline_fetch_all_requests.time()
+@metrics.api_exceptions.count_exceptions()
+def get_baselines_by_system_id(system_id=None):
+    account_number = view_helpers.get_account_number(request)
+
+    if system_id:
+        validate_uuids([system_id])
+        query = SystemBaselineMappedSystem.query.filter(
+            SystemBaselineMappedSystem.account == account_number,
+            SystemBaselineMappedSystem.system_id == system_id,
+        )
+    else:
+        query = SystemBaselineMappedSystem.query.filter(
+            SystemBaselineMappedSystem.account == account_number
+        )
+
+    query_results = query.all()
+
+    message = "read baselines with system"
+    current_app.logger.audit(message, request=request, success=True)
```

Comment:
In addition to success=True, we should set up to catch some errors in case the query fails, like we did for the other endpoints you just added.  I know that conversation is still ongoing, and you were  going to explore potentially over the weekend options for better error messages, but we should at least set up to catch something before attempting to iterate and return.

Reply:
I will wrap `query.all()` to catch any errors for now.

Reply:
Done